gaitutils/emg.py

```diff
@@ -104,7 +104,7 @@ def _edf_export(self, filename):
         # default to EDF+ for the time being
```

Comment:
If `pyedflib` is going to become a "normal" dependency, let's remove the function-level import check above and move the import to top level?

Reply:
Definitely.

Reply:
Ok. If you update your branch, I think the changes will show in this PR too